ocaml/xapi/xapi_vm.ml

```diff
@@ -991,7 +991,16 @@ let s3_suspend ~__context ~vm = Xapi_xenops.s3suspend ~__context ~self:vm
 let s3_resume ~__context ~vm = Xapi_xenops.s3resume ~__context ~self:vm
 
 let set_bios_strings ~__context ~self ~value =
-  ()
+  (* Only allow to fill in BIOS strings if they are not yet set *)
```

Comment:
This comment makes it sound as if each individual string can be set if it is not yet set, even if other strings have been set.

Perhaps it would be clearer as `(* Allowed only if the VM has no BIOS strings *)` or similar.